lib/TPP/RewriteToBatchReduceGemm.cpp

```diff
@@ -28,7 +28,7 @@ using namespace mlir;
 
 #define DEBUG_TYPE "mlir-rewrite-to-brgemm"
 
-// Look for [p ... p] brgemm [r p p r r]
+// Look for [p ... p] brgemm [r r p p r]
```

Comment:
both `checkVNNIStructure` and `checkStructure` are very similar and could be de-duplicated.